Cql/CoreTests/ExpressionBuilderTests.cs

```diff
@@ -22,7 +22,7 @@ public void AggregateQueries_1_0_0()
         {
             var elm = new FileInfo(@"Input\ELM\Test\Aggregates-1.0.0.json");
             var elmPackage = Hl7.Cql.Elm.Library.LoadFromJson(elm);
-            var definitions = Factory.LibraryDefinitionsBuilder.ProcessLibrary(elmPackage);
+            var definitions = Factory.LibraryExpressionBuilder.ProcessLibrary(elmPackage);

```

Comment:
The `LibraryDefinitionsBuilder` type was removed, so this might look like a rename, but it isn't. Agree, the term `Expression` is extremely overloaded. We should pick a better name for `LibrarySet`-, `Library`- and `ExpressionBuilder`. We can discuss.